index.html

```diff
@@ -5,8 +5,114 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <img src="logo.svg" alt="Город Седона." width="40" height="30">
+        <ul class="navigation-list">
+           <li class="navigaton-item">
+             <a class="navigation-link" href="#">Главная</a>
+           </li>
+           <li class="navigaton-item">
+             <a class="navigation-link" href="#">О Седоне</a>
+             </li>
+           <li class="navigation-item">
+             <a class="navigation-link" href="catalog.html">Гостиницы</a>
+             </li>
```

Comment:
```suggestion
           </li>
```

не до конца отбьютифаела. Надо бьютифай поставить на сейв файла

Reply:
не сделано